scraper/google.py

```diff
@@ -0,0 +1,218 @@
+import requests
+from bs4 import BeautifulSoup
+from selenium.webdriver import Firefox, FirefoxOptions
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.wait import WebDriverWait
+
+# Firefox is an alias for the WebDriver object from
+# selenium.webdriver.firefox.webdriver
+
+options = FirefoxOptions()
+options.add_argument("-headless")
+driver = Firefox(options)
+
+google_site = "https://careers.google.com/jobs/results/?distance=50&employment_type=INTERN&location=United%20States&location=Canada"
```

Comment:
How will distance apply when considering Us and Canada btw?

Reply:
honestly not sure, `distance=50` is a default parameter applied by Google when you start adding filters to their job search site

Reply:
Maybe we want to max this out or unset it/remove it?